tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
We had agreed that pool name will continue to be docker-pool if user did not pass in any input/output files. This is legacy docker mode. So if we do that, these tests should not fail.